build.proj

```diff
@@ -5,17 +5,16 @@
   <Target Name="Build" DependsOnTargets="PrepareOutput;InitBuild">
     <Message Text="Build Environment: $(Platform) $(Configuration) $(TargetOS) $(TargetRid)" />
 
-    <ItemGroup>
-      <_RootProject Include="$(ProjectDir)targets/repository.proj" Condition="'%(Repository.Identity)' == '$(DefaultRepository)'">
-        <AdditionalProperties>RepositoryName=%(Repository.Identity);PathToRepo=%(Repository.PathToRepo);RepositoryOrganization=%(Repository.Organization);RepositoryBranch=%(Repository.Branch)</AdditionalProperties>
-      </_RootProject>
-    </ItemGroup>
+    <MSBuild Projects="repos\$(RootRepo).proj" Targets="Build" BuildInParallel="$(BuildInParallel)" />
+  </Target>
 
-    <MSBuild Projects="@(_RootProject)" Targets="BuildRepositoryAndDependencies" BuildInParallel="$(BuildInParallel)" />
+  <Target Name="ApplyPatches">
+    <MSBuild Projects="repos\$(RootRepo).proj" Targets="ApplyPatches" BuildInParallel="$(BuildInParallel)" />
```

Comment:
Doesn't this just apply patches to the one repo? 

Do we really need this target?

Reply:
we could remove it but I found it helpful when just working on one repo.

Reply:
I think that it's confusing if I say

`build /t:ApplyPatches`

that it will only apply patches to the CLI repo.

Reply:
OK. Would you suggest I remove it completely or make it apply all the patches?

Reply:
Either of those 2 things would work for me.

I'd lean towards removing it.  The init-build.proj will already apply all patches the first time a build is executed.